.github/workflows/ci.yml

```diff
@@ -41,6 +154,7 @@ jobs:
       - name: Build image
         if: ${{ steps.path_diff.outputs.changed == 1 }}
         uses: docker/build-push-action@v2
+        # TODO: surely this might be related to the fork, as they might have an updated one!:
```

Comment:
Yes you are right but there is an unfortunate bug. Docker only supports lower case names as image names, so if you use the GitHub placeholders for the repo name, you end up with "DragonMinded" here (with the uppercase) which doesn't work. 

I tried looked around for a workaround but could not find one, so for now this is how it is.

Please change the TODO to report this, so that we will track it